server/index.js

```diff
@@ -132,6 +133,9 @@ if (!dev) {
   }
 }
 
+// Server-side routes that do not render the application
+app.use('/api', apiRouter);
```

Comment:
Could you also add a note to here:
https://github.com/sharetribe/ftw-daily/blob/master/src/routeConfiguration.js#L48

Just to ensure that if there's a clash of existing code in some of the customizations the first step is to look from routeConfiguration that what's wrong there. Some code comment would give a hint that people need to take a look to server/index.js too.